### PR TITLE
Add package average and ignore vendor directory

### PR DESCRIPTION
**Pull request by mohit**

### CODE REVIEW

File: gocyclo.go [hunk 1]
```diff
@@ -142,6 +163,26 @@ func average(stats []stat) float64 {
...
+	}
+}
+
+func statsByPkg(stats []stat) map[string][]stat {
+	pkgStats := map[string][]stat{}
+	for _, s := range stats {
+		if _, ok := pkgStats[s.PkgName]; ok {
+			pkgStats[s.PkgName] = append(pkgStats[s.PkgName], s)
+		} else {
+			pkgStats[s.PkgName] = []stat{}
```
Comment (jiancheung) [inline gocyclo.go]: should this be `pkgStats[s.PkgName] = []stat{s}`?

Comment (mohit) [reply gocyclo.go]: derp yes.

Comment (mohit) [reply gocyclo.go]: https://github.com/Clever/gocyclo/pull/1/commits/af4b1e001932e15896475343006d441316a226bc

**Comment by jiancheung** [comment]
lgtm!

**Review by jiancheung — APPROVED** [review]

**Pull request merged**